package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "gusto",
+  "version": "0.1.1",
+  "main": "src/api.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Stratera/gusto.git"
+  },
+  "dependencies": {
+    "express": "^4.12.3"
+  },
+  "devDependencies": {
+    "mocha": "~2.1.0",
+    "mocha-teamcity-reporter": ">=0.0.1"
+  },
+  "scripts": {
+    "test": "npm run unit-test-tc",
+    "start": "node server.js",
+    "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha ./test/unit/mocha --report html",
+    "unit-test": "mocha ./test/unit/mocha",
```

Comment:
Used mocha with good success at Gannett so it's a familiar platform for me for both unit and integration testing - jasmine looks pretty similar with some neat distinctions... may look into prior to the official round of unit tests.